src/globus_cli/commands/gcs/endpoint/show.py

```diff
@@ -0,0 +1,22 @@
+import uuid
+
+from globus_cli.commands.gcs.endpoint._common import GCS_ENDPOINT_FIELDS
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import command, endpoint_id_arg
+from globus_cli.termio import TextMode, display
+
+
+@command("show")
+@endpoint_id_arg
+@LoginManager.requires_login("transfer")
+def show_command(
+    login_manager: LoginManager,
+    *,
+    endpoint_id: uuid.UUID,
+) -> None:
+    """Describe a GCS Endpoint"""
```

Comment:
I anticipated something akin to "Show information about a GCS Endpoint". Does the "show"-command/"describe"-text match other art?

Reply:
I think it just depends on who wrote it:

**sirosen**
* [Display information about an index](https://github.com/globus/globus-cli/blob/main/src/globus_cli/commands/search/index/show.py#L14)
* [Print information detailing the status and other info about a task. The task may be pending, completed, or in progress.](https://github.com/globus/globus-cli/blob/main/src/globus_cli/commands/task/show.py#L184C5-L186C56)
* [Show a flow](https://github.com/globus/globus-cli/blob/main/src/globus_cli/commands/flows/show.py#L15)

**Derek**
* [Describe a particular role on a GCS Endpoint.](https://github.com/globus/globus-cli/blob/main/src/globus_cli/commands/gcs/endpoint/role/show.py#L21)

**Kurt**
* [Display information about a particular timer.](https://github.com/globus/globus-cli/blob/main/src/globus_cli/commands/timer/show.py#L17)

**Aaron**
* [Show a group definition](https://github.com/globus/globus-cli/blob/main/src/globus_cli/commands/group/show.py#L14)

Reply:
I am in favor of this new helptext because it apparently doesn't quite match any of the existing helptext. :upside_down_face: 

I would not want to block this change on any concern about things drifting even further apart, but if Derek (as the current PR author) wants to pick a direction and lead us all towards more uniform shorthelp for our commands, I'm more than happy to follow.

Reply:
I'm not gonna be the one to spearhead helptext wording cause I don't think I'm very good at that.

But if there's a common pattern (ideally enforced as much as possible by linter) I can follow it!

Reply:
Having said that though, I like Kurt's wording better than mine so I updated it to `Display information about a particular GCS Endpoint.`